src/event_handler.cpp

```diff
@@ -307,11 +307,18 @@ bool EventHandler::IsAuthenticated(const Event& event) {
 			|| event.get_executer().IsVerified());
 }
 
+void EventHandler::CheckChannelTarget(Event& event) {
+	const std::string& target = event.get_command_params()[0];
+	//is channel but not but cannot find channel target
+	if (*target.c_str() == '#' && !event.IsChannelEvent()) {
```

Comment:
ちょっと分かりにくいので、
```suggestion
	if (target[0] == '#' && !event.IsChannelEvent()) {
```
こんなのはどうでしょうか。
operator[]はconstの場合も対応しているようです。
[リファレンス](https://cpprefjp.github.io/reference/string/basic_string/op_at.html)

Reply:
その通りです、その方がわかりやすいですので修正を承認します